src/test/java/it/gov/pagopa/reward/notification/rest/UserRestClientImplTestIntegrated.java

```diff
@@ -0,0 +1,57 @@
+package it.gov.pagopa.reward.notification.rest;
+
+import it.gov.pagopa.reward.notification.BaseIntegrationTest;
+import it.gov.pagopa.reward.notification.dto.rest.UserInfoPDV;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.support.TestPropertySourceUtils;
+import org.springframework.web.reactive.function.client.WebClientException;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+
+@SuppressWarnings("squid:S3577") // suppressing class name not match alert
+@ContextConfiguration(initializers = UserRestClientImplTestIntegrated.AddPDVProperties.class)
+class UserRestClientImplTestIntegrated extends BaseIntegrationTest {
+
+    @Autowired
+    private UserRestClient userRestClient;
+
+    @Value("${app.pdv.userIdOk}")
```

Comment:
Give a default value in order to suggest a possible value
```suggestion
    @Value("${app.pdv.userIdOk:EXAMPLE_DEFAULT_VALUE}")
```